packages/preview/casual-szu-report/0.1.0/typst.toml

```diff
@@ -0,0 +1,19 @@
+[package]
+name = "casual-szu-report"
+version = "0.1.0"
+entrypoint = "lib.typ"
+authors = ["Keaton Jiang <jiang131072@gmail.com>"]
+license = "MIT"
+description = "A template for SZU course reports."
+
+homepage = "https://typst.app/universe/package/casual-szu-report"
```

Comment:
This is not the intended function of the homepage field. Every package has this, so if there is no specific external homepage, just leave this out.